workos/directory_sync.py

```diff
@@ -131,6 +131,23 @@ def get_user(self, user):
 
         return WorkOSDirectoryUser.construct_from_response(response).to_dict()
 
+    def get_primary_email(self, user):
```

Comment:
We don't need this method on the service class itself.

The `get_primary_email` method on the `WorkOSDirectoryUser` is an instance method that the developer can call on the directory users returned from the other methods (like `get_user`).

Reply:
Got it, will work on refactoring this.